src/app.py

```diff
@@ -363,24 +363,46 @@ def schedule_surgery(hospitalization_id):
 def get_prescriptions(person_id):
     logger.info(f'GET {request.path}')
 
-    token = get_jwt()
-    identity = get_jwt_identity()
-    payload = request.get_json()
+    # 1. get token data
+    id = get_jwt_identity()
+    type = get_jwt().get('type')
+
+    # 2. check if patient is target patient
+    if type == 'patient':
+        if id != person_id:
```

Comment:
aqui é preferivel utilizar o enum e para n ter 2 if nested eu substituiria com um and